package.json

```diff
@@ -133,6 +136,7 @@
     "jest": "27.4.5",
     "libsodium": "0.7.8",
     "libsodium-wrappers": "0.7.8",
+    "localstorage-migrator": "^1.0.8",
```

Comment:
IMHO, relying on quite unpopular and opinionated libraries sounds way worse than creating your own small solutions. Let's implement the migrator ourselves as this library is not maintained and not widely used